README.md

```diff
@@ -265,3 +254,23 @@ If the received value is `3`, then it will be printed as `03`.
 [ci-runs]:             https://github.com/golang/mock/actions
 [reference-badge]:     https://pkg.go.dev/badge/github.com/golang/mock.svg
 [reference]:           https://pkg.go.dev/github.com/golang/mock
+
+## Debugging Errors
+
+### reflect vendoring error
+
+```text
+cannot find package "."
+... github.com/golang/mock/mockgen/model
+```
+
+If you come across this error while using reflect mode and vendoring
+dependencies there are three workarounds you can choose from:
+
+1. Use source mode.
+2. Include an empty import `import _ "github.com/golang/mock/mockgen/model"`.
+3. Add `--build_flags=--mod=mod` to your mockgen command.
+
+This error is do to changes in default behavior of the go command in more recent
```

Comment:
Typo: s/do/due/

Reply:
Whoops, thank you for catching this! I will get it updated right away!

Reply:
https://github.com/golang/mock/pull/568